src/audio/eq_fir/eq_fir.c

```diff
@@ -133,6 +134,73 @@ static inline int set_fir_func(struct processing_module *mod, enum sof_ipc_frame
 	}
 	return 0;
 }
+#endif /* CONFIG_IPC_MAJOR_3 */
+
+#if CONFIG_IPC_MAJOR_4
+static inline int set_fir_func(struct processing_module *mod, enum sof_ipc_frame fmt)
+{
+	struct comp_data *cd = module_get_private_data(mod);
+	unsigned int valid_bit_depth = mod->priv.cfg.base_cfg.audio_fmt.valid_bit_depth;
+
+	comp_info(mod->dev, "set_fir_func(): valid_bit_depth %d", valid_bit_depth);
```

Comment:
do we need this? I understand that this is only used during setup, but it's also possible that we're setting up this stream while other streaming is already running

Reply:
I guess not if it it works OK. Before the IPC4 changes a wrong function got selected. Change this to comp_dbg()?

Reply:
Done